.clang-format

```diff
@@ -1,8 +1 @@
 BasedOnStyle: LLVM
-UseTab: Never
-IndentWidth: 4
-TabWidth: 4
-AccessModifierOffset: -2
-BreakBeforeBraces: Stroustrup
-FixNamespaceComments: true
-ColumnLimit: 100
```

Comment:
@mlxd we discuss this yesterday and here is a fix for the code style inconsistency. What do you think?

Reply:
`BreakBeforeBraces: Stroustrup`  There's the culprit. Sure, sounds good to me

Reply:
Yep, there it is! :)